README.md

```diff
@@ -18,7 +18,7 @@ The ultimate goal of this project is to allow Minecraft: Bedrock Edition users t
 
 Special thanks to the DragonProxy project for being a trailblazer in protocol translation and for all the team members who have now joined us here!
 
-### Currently supporting Minecraft Bedrock v1.16.100/v1.16.101/v1.16.200 and Minecraft Java v1.16.4.
+### Currently supporting Minecraft Bedrock v1.16.100/v1.16.101/v1.16.200/v1.16.201 and Minecraft Java v1.16.4.
```

Comment:
At this point, it might be better separated by commas? `v1.16.100, v1.16.101, v1.16.200, v1.16.201`.